sdk/test-utils/recorder-new/src/sanitizer.ts

```diff
@@ -140,9 +142,10 @@ export class Sanitizer {
    * - generalRegexSanitizer is applied for each of the parts with the real and fake values that are parsed
    */
   async addConnectionStringSanitizer(
-    actualConnString: string,
+    actualConnString: string | undefined,
     fakeConnString: string
   ): Promise<void> {
+    if (!actualConnString) return;
```

Comment:
should we throw if it's undefined in live/record mode?

Reply:
done
only record mode**